api/files/views.py

```diff
@@ -87,6 +87,9 @@ def get_node(self):
     def get_object(self):
         user = utils.get_user_auth(self.request).user
         file = self.get_file()
+        if file.node.is_quickfiles and not file.node.creator.is_active:
```

Comment:
I think this logic belongs in `FileMixin`'s `get_file()`. Having it here prevents the file detail endpoint from being accessed, but not the file versions endpoint.